src/Format.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Spatie\Feed;
+
+class Format
+{
+    /*
+    * Escape given sequence to remove ending CDATA tags
+    *
+    * @param string $value
+    * @return string
+    */
+    public static function escape_cdata($value)
```

Comment:
No need to have this snaked_cased anymore. Rename this to it's camelCased variant